internal/exec/util/blkid.c

```diff
@@ -248,3 +254,39 @@ static result_t extract_part_info(blkid_partition part, struct partition_info *i
 
 	return RESULT_OK;
 }
+
+// blkid_get_block_devices fetches block devices with the given FSTYPE
+result_t blkid_get_block_devices(const char *fstype, struct block_device_list *device) {
+	blkid_dev_iterate iter _cleanup_iterator_ = NULL;
+	blkid_dev dev;
+	blkid_cache cache _cleanup_cache_ = NULL;
+	int err, count = 0;
+	const char *ctmp = NULL;
+	
+	if (blkid_get_cache(&cache, "/dev/null") != 0)
+	   return RESULT_GET_CACHE_FAILED;
```

Comment:
There's still a lot of inconsistent indentation here.  In this file, one level is one tab.

Reply:
Does it look good now?

Reply:
Yup, thanks.